urbanbroccoli/laplacian.py

```diff
@@ -10,16 +10,33 @@ def create_laplacian_1d(nx, hx):
     return laplacian
 
 def create_laplacian_2d(nx, ny, hx, hy):
-    laplacian = zeros(shape=(nx, ny, nx, ny), dtype=float64)
+    '''Creates Laplacian matrix for 2-D Domain by discretizing the domain by grids of required sizes in X and Y direction.  
```

Comment:
This line is way too long. Please shorten to just a few words which describe as basic as possible what is going on.

Reply:
Changes done.